samples/templates/express-embedded-auth-with-sdk/web-server/views/authenticator-webauthn-enroll.mustache

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    {{>head}}
+    <script type="text/javascript" src="/assets/js/webauthn.js"></script>
+  </head>
+  <body id="samples">
+  
+    {{>menu}}
+
+    <div class="okta-sign-in-form ui center aligned relaxed grid">
+      <div class="row">
+        <div id="page-title-header" class="ui header">
+          <h2>{{title}}</h2>
+        </div>
+      </div>
+
+      <div class="row">
+        {{>formMessages}}
+      </div>
+
+      {{#activationData}}
+        <div class="row">
+          <form id="challenge-authenticator-form" class="ui large form" action="{{action}}" method="POST">
+
+            <input id="credentials-id" type="hidden" name="id" autoComplete="off" required />
+            <input id="credentials-clientData" type="hidden" name="clientData" autoComplete="off" required />
+            <input id="credentials-attestation" type="hidden" name="attestation" autoComplete="off" required />
+            
+            <div class="row">
+              <div id="credentials-error" class="ui negative message"></div>
+            </div>
+
+            <div class="row">
+              <button id="retry-button" class="ui fluid large button" onclick="askCredentials()" type="button">
+                Retry
+              </button>
+
+              <button id="submit-button" class="ui fluid large primary submit button" type="submit">
+                Continue
+              </button>
+            </div>
+          </form>
+        </div>
+
+        <script type="text/javascript">
+          const activationData = {{{activationData}}};
+          const authenticatorEnrollments = {{{authenticatorEnrollments}}};
+
+          const askCredentials = async () => {
+            try {
+              const options = buildCredentialCreationOptions(activationData, authenticatorEnrollments);
+              const credential = await CredentialsContainer.create(options);
+              const res = getAttestation(credential);
```

Comment:
Can it be calculated from server side?

Reply:
Then we need to send `Uint8Array`s (`clientDataJSON` and `attestationObject` are [array buffers](https://developer.mozilla.org/en-US/docs/Web/API/AuthenticatorAttestationResponse)) to server with proper encoding. Looks like best way for this is base64 encoding which is already handled in `getAttestation `

Maybe we want to put methods
- `buildCredentialCreationOptions `
- `buildCredentialRequestOptions`
- `getAttestation`
- `getAssertion`
in SDK to hide all that conversions from developer?

Or create only 2 wrapper methods to get attestation (for enroll) and assertion (for verify), wrapping work with webauthn. 
These 2 method signature would look like this:
- getAttestation(activationData, authenticatorEnrollments) -> {clientData, attestation}
- getAssertion(challengeData, authenticatorEnrollments) -> {clientData, authenticatorData, signatureData}

(work with webauthn is different for Edge, see [polyfill](https://github.com/adrianba/fido-snippets/blob/master/polyfill/webauthn.js))